components/motor/tmcstepper/stepper_motor_tmc.go

```diff
@@ -63,6 +63,9 @@ func (config *TMC5072Config) Validate(path string) ([]string, error) {
 	if config.Index <= 0 {
 		return nil, resource.NewConfigValidationFieldRequiredError(path, "index")
 	}
+	if config.Index > 2 {
+		return nil, errors.New("tmcstepper motor index should be 1 or 2")
+	}
```

Comment:
q: just curious, why is it now 1 & 2 instead of 0 & 1?

Reply:
0 never actually worked because of the check above this